### DISCUSSION TITLE
Add css file

### DISCUSSION THREAD

**Discussion by ctuxboy** [BoidCMS/BoidCMS#42] [Q&A] [answered]
Hello,
When looking in the docs i can't found a solution how adding a local css file to my theme.

**../mytheme/css/style.css**

The BoidCMS theme examples doesn't have a link to a local css file.

I try this:

`<link rel="stylesheet" href="<?php echo $App->get( 'url' ) ?>mytheme/css/style.css">`

With no luck :(

Someone knows how add a theme's css-file?

**Comment by ctuxboy** [comment, marked as answer]
Found the solution in the [Noir theme](https://github.com/BoidCMS/Noir/blob/master/parts/head.php)!

`<link rel="preload stylesheet" href="<?= $App->theme( 'css/style.css', false ) ?>">`

**Comment by sysadev** [comment]
There is a documentation of the `App` class here https://boidcms.github.io/#/developer/reference. There you could also find all about the `App::theme` method and its parameters.

https://github.com/BoidCMS/boidcms.github.io/blob/821a2357e514a72493ac6501bf43819e5939da7d/developer/reference.md?plain=1#L191-L200
